missions/FUJ_Missions.txt

```diff
@@ -1144,26 +1483,8 @@ fujiwara_missions_keine_slot = {
 				}
 			}
 			add_stability_or_adm_power_per_stab = { amount = 1 }
-			Bamboo_forest_of_the_lost = {
-				limit = {
-					owned_by = ROOT
-				}
-				add_province_modifier = {
-					name = fuj_mokou_administration
-					duration = 3650
-				}
-				add_local_autonomy = -30 
-			}
-			Eientei_state = {
-				limit = {
-					owned_by = ROOT
-				}
-				add_province_modifier = {
-					name = fuj_mokou_administration
-					duration = 3650
-				}
-				add_local_autonomy = -30 
-			}
```

Comment:
Why the decision to remove these modifiers?

Reply:
apologizes what i ment was that the states no longer exist as in the title is no longer in use, so any call to them would break the mission tree